cfg/config_test.go

```diff
@@ -9,15 +9,17 @@ import (
 func TestNew(t *testing.T) {
 	conf, err := New("/foobar")
 	if err != nil {
-		panic(err)
+		t.Error("cannot create config from path")
```

Comment:
Couldn't we just use `t.Fatal` here?

Reply:
Thank you! I'm a bit lost without assertify :upside_down_face: Fixed